src/LiveComponent/src/Resources/views/form_theme.html.twig

```diff
@@ -0,0 +1,35 @@
+{%- block live_collection_widget -%}
+    {% set attr = attr|merge({'class': attr.class|default('space-y-4')}) %}
+    <div {{ block('widget_container_attributes') }}>
+        {%- if form is rootform -%}
+            {{ form_errors(form) }}
+        {%- endif -%}
+        {{- block('form_rows') -}}
+        {{- form_rest(form) -}}
+
+        {% if button_add_prototype is defined and not button_add_prototype.rendered %}
+            {{ form_row(button_add_prototype, { attr: button_add_prototype.vars.attr|merge({
+                'data-action': 'live#action',
+                'data-action-name': 'addCollectionItem(name=' ~ form.vars.full_name ~ ', index=' ~ (form.children is empty ? 0 : max(form.children|keys)) ~ ')'
+            }) }) }}
+        {% endif %}
+    </div>
+{%- endblock live_collection_widget -%}
+
+{% block live_collection_entry_row %}
+    {{ block('form_row') }}
+    {% if button_delete_prototype is defined and not button_delete_prototype.rendered %}
+        {{ form_row(button_delete_prototype, { attr: button_delete_prototype.vars.attr|merge({
+            'data-action': 'live#action',
+            'data-action-name': 'removeCollectionItem(name=' ~ form.parent.vars.full_name ~ ', index=' ~ form.vars.name ~ ')'
+        }) }) }}
+    {% endif %}
+{% endblock live_collection_entry_row %}
+
+{% block live_collection_button_add_widget %}
+    {{ block('button_widget') }}
+{% endblock live_collection_button_add_widget %}
```

Comment:
This should be removed by default. Just added here to highlight we have block for the buttons, so we can customize as we like, e.g.:

```twig
{% block live_collection_button_add_widget %}
    {% set attr = attr|merge({'class': attr.class|default('btn btn-ghost')}) %}
    {% set translation_domin = false %}
    {% set label_html = true %}
    {%- set label -%}
        <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
            <path stroke-linecap="round" stroke-linejoin="round" d="M12 6v6m0 0v6m0-6h6m-6 0H6" />
        </svg>
        {{ 'form.collection.button.add.label'|trans({}, 'forms') }}
    {%- endset -%}
    {{ block('button_widget') }}
{% endblock live_collection_button_add_widget %}
```